cmd/root.go

```diff
@@ -122,6 +122,8 @@ any client SSL certificates.`,
 		"Initial port to use for listeners. Subsequent listeners increment from this value.")
 	cmd.PersistentFlags().StringVarP(&c.conf.UnixSocket, "unix-socket", "u", "",
 		`Enables Unix sockets for all listeners using the provided directory.`)
+	cmd.PersistentFlags().BoolVarP(&c.conf.IAMAuthN, "iam-authn-login", "i", false,
```

Comment:
how about "--auto-iam-authn"?

It could maybe even be "auto-iam" or "auto-authn" but they feel a bit too short.

Reply:
auto-iam-authn is good -- it matches the feature and is easy to remember.